ignite/contrib/handlers/hpo_logger.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+
+import collections
+from collections import defaultdict, UserString
+from collections.abc import Mapping, Sequence
+from enum import Enum
+import json
+import os
+from threading import RLock
+import time
+from typing import Any
+
+from ignite.engine import Engine, Events
+
+_DEFAULT_METRIC_PATH = '/tmp/hypertune/output.metrics'
```

Comment:
I wonder whether this code could run on windows. If yes, we may need to provide OS-agnostic path using python tempfile module.

Reply:
this is the path where the listener of the GCP HPO orchestration has a look. Whatever is dumped here (in terms of metric values) becomes visible to the Vertex platform. I found this path here https://github.com/GoogleCloudPlatform/cloudml-hypertune/blob/8e3530e1c4926ac64cf28330d1104b838e07a468/hypertune/hypertune.py#L24

Should we keep it?